docs/vaadin-router/demo/vaadin-router-code-splitting-demos.html

```diff
@@ -24,8 +24,8 @@ <h3>Lazy Loading JS Bundles</h3>
           ```js
           {
             bundle: {
-              src: '/chunk.js', // required, contains the bundle URL
-              type: 'module' // optional, default to "nomodule"
+              module: '/bundle.esm.js', // for browsers supporting ES modules
+              type: '/bundle.nomodule.js' // fallback for older browsers
```

Comment:
Should this be `nomodule` instead of `type`?

Reply:
Good catch, thanks!